### PR TITLE
R datascience book

### PR DESCRIPTION
**Pull request by baogianghoangvu**
Add "Data Analysis and Prediction Algorithms with R" by Rafael A. Irizarry

## What does this PR do?
Add Resource(s) 

### Description 
"Data Analysis and Prediction Algorithms with R" by Rafael A. Irizarry

### Why is this valuable (or not)
The Datascience in R section is still quite sparse. This book is one of the best material for newcomers to datascience in R.

### How do we know it's really free?
Freely available in included URL and from author's Github page.

### For book lists, is it a book?
A book

### Checklist:
- [ x] Not a duplicate
- [ x] Included author(s) if appropriate
- [ x] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -2369,6 +2369,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [An Introduction to Statistical Learning with Applications in R](http://www-bcf.usc.edu/~gareth/ISL/) - Gareth James, Daniela Witten, Trevor Hastie and Robert Tibshirani (PDF)
 * [Cookbook for R](http://www.cookbook-r.com) - Winston Chang
 * [Data Analysis for the Life Sciences](https://leanpub.com/dataanalysisforthelifesciences) - Rafael A Irizarry, Michael I Love *(Leanpub account or valid email requested)*
+* [Data Analysis and Prediction Algorithms with R](https://rafalab.github.io/dsbook/) - Rafael A. Irizarry
 * [Exploratory Data Analysis with R](https://leanpub.com/exdata) - Roger D. Peng *(Leanpub account or valid email requested)*
```
Comment (Swapnil123123) [inline free-programming-books.md:2373]: maintain alphabetical order in order for the checktto pass

Comment (baogianghoangvu) [reply free-programming-books.md:2373]: Oh thank you for the review. I missed that.

**Comment by eshellman** [comment]
Thanks and welcome to free-programming-books!

@Swapnil123123 thanks for the assist!

**Comment by bigbirdone** [comment]
3891

在 2020-10-12 10:08:35，"Eric Hellman" <notifications@github.com> 写道：


Merged #3891 into master.

**Pull request merged**